README.md

```diff
@@ -21,17 +18,114 @@ To toggle between the Phase 5 transition and post-transition modes we have defin
 #### Transition
 <pre>
 sm2 --start CTC_TRADERS_P5_ACCEPTANCE_TRANSITION
-sm2 --stop CTC_DEPARTURE_TRADER_DETAILS_FRONTEND_TRANSITION
+sm2 --stop MANAGE_TRANSIT_MOVEMENTS_DEPARTURE_FRONTEND_TRANSITION
 sbt -Dplay.additional.module=config.TransitionModule run
 </pre>
 
 #### Final
 <pre>
 sm2 --start CTC_TRADERS_P5_ACCEPTANCE
-sm2 --stop CTC_DEPARTURE_TRADER_DETAILS_FRONTEND
+sm2 --stop MANAGE_TRANSIT_MOVEMENTS_DEPARTURE_FRONTEND
```

Comment:
Can we leave this as the relevant service

Reply:
Yep, I missed that when copying over previous changes